api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/MessageTypeProcessor.java

```diff
@@ -22,4 +23,15 @@
      */
     MessageType supportsMessageType();
 
+    /**
+     * Can be implemented by a MessageTypeProcessor to define custom functionality that needs to be performed
+     * when the processing of a message has failed. Default functionality as defined here is to do nothing.
+     *
+     * @param message      The message that has failed
+     * @param failureEvent The details of the failed event.
+     */
+    @SuppressWarnings("PMD.UncommentedEmptyMethodBody")
```

Comment:
Its annoying I had to add this because its JavaDoc'd but PMD can't see it for some reason.

Reply:
We can remove the `@SuppressWarnings` and add a `// No action required` comment in the method body itself.

Reply:
Removed suppress warnings.